scrooge-generator/src/main/resources/scalagen/finagleServiceFunction.scala

```diff
@@ -15,6 +22,12 @@ addFunction("{{serviceFuncNameForWire}}", { (iprot: TProtocol, seqid: Int) =>
       }
 {{/exceptions}}
       case e => Future.exception(e)
+    } respond {
```

Comment:
we've been switching over lazily–if you could do a bunch, it would be great, but it's ok to just switch over lazily too.